2.0/db_models/toc_section_embds.py

```
@@ -0,0 +1,40 @@
+                            index=True)
+    '''
+                            
+    # Foreign key to toc_sections 
+    section_id = Column(Integer, 
+                       ForeignKey(f'{settings.TOC_SECTIONS_TABLE}.id'),
+                       nullable=True,
+                       index=True)
+    
+    # The vector embedding (384 dimensions for all-MiniLM-L6-v2)
+    embedding = Column(Vector(384),  # pgvector column type
+                      nullable=False)
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)